modules/opm-cli-ref-serve.adoc

```diff
@@ -23,9 +23,21 @@ $ opm serve <source_path> [<flags>]
 |===
 |Flag |Description
 
+|`--cache-dir` (string)
+|If set, sync and persist server cache directory.
+
+|`--cache-enforce-integrity`
+|Exit with error if cache is not present or has been invalidated. (Default: true when `--cache-dir` is set and `--cache-only` is false, false otherwise)
```

Comment:
```suggestion
|Exit with error if cache is not present or has been invalidated. The default is `true` when `--cache-dir` is set and `--cache-only` is `false`; the default is `false` otherwise.
```